frontend/components/Icons/LinkedinIcon.vue

```diff
@@ -9,7 +9,8 @@ defineProps({
 
 <template>
   <a :href="href" class="transition duration-200 hover:scale-110 hover:text-[#0A66C2]">
-    <svg id="Icons" fill="currentColor" height="35px" width="35px" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" xml:space="preserve">
+    <span class="sr-only">Linkedin</span>
+    <svg id="Icons" fill="currentColor" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" xml:space="preserve">
```

Comment:
I think id should be more descriptive, something like "LinkedIn".

Reply:
right, I just commited this change.